numpy/core/numerictypes.pyi

```diff
@@ -118,11 +118,6 @@ def issubdtype(arg1: DTypeLike, arg2: DTypeLike) -> bool: ...
 
 def sctype2char(sctype: DTypeLike) -> str: ...
 
-def find_common_type(
```

Comment:
Is the typing info removed now or when the deprecation expires?

Reply:
I think the decision was that typing lives in the future because typing doesn't really support deprecations, ping @BvB93 .

Reply:
Generally speaking we remove the annotations the moment the deprecation goes live, as this will cause type checkers to emit warnings similar as done during runtime with the likes of `DeprecationWarning`. It might be useful to add something akin to a `# Deprecated: find_common_type` comment though for future reference.

An another note, there is still a remaining `find_common_type` re-export in the main stub file: https://github.com/numpy/numpy/blob/9a6d956b273fe8f4d3ecd1d1f08f8a1c2e4edc06/numpy/__init__.pyi#L396

Reply:
The change to not re-export `find_common_type` in `__init__.pyi` still needs doing

Reply:
@seberg Note that `__init__.pyi` still needs fixing

Reply:
I made the change to `__init__.pyi`.